wallet/src/main/scala/org/bitcoins/wallet/models/SpendingInfoDAO.scala

```diff
@@ -483,7 +482,8 @@ case class SpendingInfoDAO()(implicit
 
     def scriptWitnessOpt: Rep[Option[ScriptWitness]] = column("script_witness")
 
-    def blockHash: Rep[Option[DoubleSha256DigestBE]] = column("block_hash")
+    def spendingTxIdOpt: Rep[Option[DoubleSha256DigestBE]] = column(
+      "spending_tx_id")
```

Comment:
nit: Let's try to be consistent with how we do `txid` Above we use `txid` but we are using `tx_id`. 

For those of us that use sql queries this with slowly drive us mad.  Consistency gives a much better developer/devops experience :-)